content/01.main_content.md

```diff
@@ -1,3 +1,15 @@
+Recognition by peers or the general public can greatly affect the trajectory of a scientist’s career.
+  Whether the recognition is the inclusion of their name in print as an expert, an invited lecture, or an award, each form of recognition paves the way to future accolades.
+  Regardless of an individual’s merit, biases can skew which scientists are even considered for recognition.
+  To combat biases, auditing systems can identify if specific groups are being unintentionally neglected and hold the recognizing body accountable.
+
+The use of auditing methods has produced evidence that they can help reduce disparities.
+  In science journalism, Adrienne LaFrance, supported by Nathan Matias, and Ed Yong performed self-audits to quantify gender disparity in their quoted or mentioned sources [@https://www.theatlantic.com/technology/archive/2016/02/gender-diversity-journalism/463023/; @https://www.theatlantic.com/science/archive/2018/02/i-spent-two-years-trying-to-fix-the-gender-imbalance-in-my-stories/552404/].
+  While a secondary audit performed by LaFrance showed no improvement from her initial audit, Yong, through continuous self-auditing, was able to achieve gender parity in his reporting.
+
+In award recognition, an audit of the International Society for Computational Biology (ISCB) honorees revealed a significant disparity against people with East Asian name origins and towards US-affiliated scientists.
+  After this study was made available to the public, the following set of honorees had the highest mean predicted probability of an honoree having an East Asian name of any previous year.
+  Additionally, it inducted the first China-based fellow into ISCB [@doi:10.1016/j.cels.2021.07.007].
```

Comment:
I'd restate as "Additionally, ISCB inducted its first China-based fellow that year" since it's unclear what "it" is in this sentence. Alternately you could say "Additionally, the nominating committee inducted the first China-based fellow into ISCB."

Reply:
yes, much better https://github.com/greenelab/disparities_opinion/pull/3/commits/63e9c31d513d7cbe15b58da051bb4281a11c2f4d